### PR TITLE
feat: 🎸 add ocsp responder to revocation type

### PR DESCRIPTION
**Pull request by waynewee**
Added new RevocationType OCSP_RESPONDER to allow for OCSP based
revocation

### CODE REVIEW

File: src/shared/utils/diagnose.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ const diagnoseV2 = ({ kind, document, debug, mode }: { kind: Kind; document: any
   try {
     // eslint-disable-next-line @typescript-eslint/no-unused-expressions
     mode === "strict" ? SignatureStrict.check(document.signature) : Signature.check(document.signature);
-  } catch (e) {
+  } catch (e: any) {
```
Comment (Nebulis) [inline src/shared/utils/diagnose.ts:117]: Can use the genetic error.i think message is available

Comment (waynewee) [reply src/shared/utils/diagnose.ts:117]: Catch clause variable type annotation must be 'any' or 'unknown' if specified.

https://github.com/Microsoft/TypeScript/issues/8677#issuecomment-220385124

Comment (Nebulis) [reply src/shared/utils/diagnose.ts:117]: ok, I thought Error was allowed, but it's true an error can be a string. Let's use unknown => https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables

File: src/shared/utils/diagnose.ts [hunk 2]
```diff
@@ -114,17 +114,17 @@ const diagnoseV2 = ({ kind, document, debug, mode }: { kind: Kind; document: any
   try {
     // eslint-disable-next-line @typescript-eslint/no-unused-expressions
     mode === "strict" ? SignatureStrict.check(document.signature) : Signature.check(document.signature);
-  } catch (e: any) {
-    return handleError(debug, e.message);
+  } catch (e) {
+    if (e instanceof Error) return handleError(debug, e.message);
```
Comment (Nebulis) [inline src/shared/utils/diagnose.ts:118]: no else? :D

**Review by Nebulis — APPROVED** [review]

**Pull request merged**